components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/application/management/v1/ClientAuthenticationConfiguration.java

```diff
@@ -66,6 +67,30 @@ public void setTokenEndpointAuthSigningAlg(String tokenEndpointAuthSigningAlg) {
         this.tokenEndpointAuthSigningAlg = tokenEndpointAuthSigningAlg;
     }
 
+    /**
+     * Allow reuse of the private key for JWT generation at the token endpoint.
```

Comment:
Are we reusing private key jwt or JTI?

Reply:
In either case it is a jti that is getting reused. Though we use the same JWT the claims are getting validated anyways. Such as exp and nbf fields. So it is the jti that's getting validated when reusing the JWT. In the spec also, they are talking about reuse of JWT and use JTI to validate.

Reply:
Anyhow this comment should be altered. AFAIR its was an autogenerated one missed to review.

Reply:
Fixed with 7d709f43b8476bb0f07718e24d2d55d4798a963b